Chargebee/Classes/Result/CBError.swift

```diff
@@ -27,18 +29,10 @@ extension CBError: LocalizedError {
             return errorResponse.message
         case .paymentFailed(let errorResponse):
             return errorResponse.message
+        case .serverError(let errorResponse):
+            return errorResponse.message
         }
     }
-    public var httpStatusCode: Int {
-        switch self {
-        case .operationFailed(let errorResponse):
-            return errorResponse.httpStatusCode
-        case .invalidRequest(let errorResponse):
-            return errorResponse.httpStatusCode
-        case .paymentFailed(let errorResponse):
-            return errorResponse.httpStatusCode
-        }
-    } 
 }
```

Comment:
Do we no longer need to expose this? 
This is public currently and hence would need to be conveyed accordingly.

Reply:
We need this reverted back thanks